.github/workflows/firebase-hosting-merge.yml

```diff
@@ -0,0 +1,30 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
+'on':
+  push:
+    branches:
+      - master
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm ci && npm run build
```

Comment:
Do we also need to include the environment variables here like we did in the PR workflow?

Reply:
We'll find out, but I don't think that was the issue